src/celeritas/phys/Interaction.hh

```diff
@@ -36,6 +36,7 @@ struct Interaction
         absorbed,  //!< Absorbed or transformed to another particle type
         unchanged,  //!< No state change, no secondaries
         failed,  //!< Ran out of memory during sampling
+        onloaded,  //!< Transfer interaction to the host
```

Comment:
I think this should go before `unchanged`: the `changed()` method below returns true if `action < Action::unchanged`, and we use this to return early from the `InteractionApplier`.

Reply:
Move it before unchanged - I did not know that the ordering matters.